policy/build_task/build_labels/build_labels.rego

```diff
@@ -5,7 +5,7 @@
 #   Policies to verify that a Tekton build task definition has the
 #   required build type label.
 #
-package build_task.labels
+package build_labels
```

Comment:
As mentioned in the description, this is a breaking change. That should be ok as, likely, the only place that uses it is [here](https://github.com/konflux-ci/build-definitions/blob/main/policies/build-tasks.yaml), and that references the bundle, not the package itself.